extensions/cornerstone/src/panels/PanelMeasurement.tsx

```diff
@@ -1,24 +1,30 @@
-import React, { useEffect, useRef, useState } from 'react';
+import React, { useEffect, useRef } from 'react';
 import { utils } from '@ohif/core';
 import { MeasurementTable } from '@ohif/ui-next';
 import debounce from 'lodash.debounce';
 import { useMeasurements } from '../hooks/useMeasurements';
+import { StudySummaryFromMetadata } from '../components/StudySummaryFromMetadata';
 
-const { filterAdditionalFinding, filterOr, filterAny } = utils.MeasurementFilters;
+const { groupByStudy } = utils.MeasurementGroupings;
+const { filterAdditionalFinding, filterAny } = utils.MeasurementFilters;
 
 export type withAppAndFilters = withAppTypes & {
   measurementFilters: Record<string, (item) => boolean>;
+  groupingFunction?: (groupedMeasurements: Map<string, object[]>, item) => Map<string, object[]>;
+  title: string;
 };
 
 export default function PanelMeasurementTable({
   servicesManager,
   commandsManager,
   customHeader,
   measurementFilters = { measurementFilter: filterAny },
```

Comment:
Could you fix this type - make it just pass in measurementFilter, it doesn't need to be multiple instances here.